src/BulletCollision/CollisionDispatch/btCollisionObjectWrapper.h

```diff
@@ -27,13 +27,19 @@ struct btCollisionObjectWrapper
 	const btCollisionShape* m_shape;
 	const btCollisionObject* m_collisionObject;
 	const btTransform& m_worldTransform;
+    const btTransform* m_preTransform;
```

Comment:
I wonder if this is really needed to add m_preTransform. May need to discuss

Reply:
I didn't see a good way around that. We should discuss.